libs/template/templates/default-python/template/{{.project_name}}/tests/main_test.py.tmpl

```diff
@@ -0,0 +1,5 @@
+from {{.project_name}} import main
+
+def test_main():
+    taxis = main.get_taxis()
+    assert len(taxis) == 5
```

Comment:
Is this test supposed to use Databricks Connect to run successfully, or is the intention to run remotely?

Reply:
This relies on Databricks Connect